### PR TITLE
Freeplane.Freeplane version 1.9.3

### PR DESCRIPTION
**Pull request by chausner**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16608)

### CODE REVIEW

File: manifests/f/Freeplane/Freeplane/1.9.3/Freeplane.Freeplane.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate
+
+PackageIdentifier: Freeplane.Freeplane
+PackageVersion: 1.9.3
+PackageLocale: en-US
+Publisher: The Freeplane team
```
Comment (ItzLevvie) [inline manifests/f/Freeplane/Freeplane/1.9.3/Freeplane.Freeplane.locale.en-US.yaml:6]:
```suggestion
Publisher: Open source
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see my review.

File: manifests/f/Freeplane/Freeplane/1.9.3/Freeplane.Freeplane.locale.en-US.yaml (hunk 1 above)
Comment (chausner) [reply manifests/f/Freeplane/Freeplane/1.9.3/Freeplane.Freeplane.locale.en-US.yaml]: Is this needed for proper correlation with Add/Remove program entries?

Comment (ItzLevvie) [reply manifests/f/Freeplane/Freeplane/1.9.3/Freeplane.Freeplane.locale.en-US.yaml]: Yes.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**